cli/cmd/config.go

```diff
@@ -48,8 +48,11 @@ func setupCommand(options ...setupOption) func(cmd *cobra.Command, args []string
 
 		options := []actions.ResourceArgsOption{actions.WithClient(apiClient), actions.WithLogger(cliLogger), actions.WithConfig(cliConfig)}
 		configActions := actions.NewConfigActions(options...)
+		pollingActions := actions.NewPollingActions(options...)
 		resourceRegistry.Register(actions.SupportedResourceConfig, configActions)
 
+		resourceRegistry.Register(actions.SupportedResourcePollingProfile, pollingActions)
```

Comment:
Having constants for supported resources defined in the `actions` package makes this package a central dependency. You can just use regular strings defined as constants within each resource package (or file, whatever applies).

Keep in mind the usecase. no code is actually consuming these constants, only when the resource registers itself. There is no other direct reference to this constant, so it's only creating a hardcoded value that is not actually consumed

Reply:
Can I update this in the following PR? it should be about adding support for the demo resource